addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -972,5 +972,77 @@ odoo.define('website.tour.form_editor', function (require) {
         },
     ]);
 
+    // Check that the editable form content is actually editable.
+    tour.register("website_form_editable_content", {
+        test: true,
+        url: "/",
+    }, [{
+            content: "Enter edit mode",
+            trigger: "a[data-action=edit]",
+        },
+        {
+            content: "Add a new 'Form' snippet",
+            trigger: "#oe_snippets .oe_snippet[name=Form] .oe_snippet_thumbnail",
+            run: "drag_and_drop #wrap",
+        },
```

Comment:
Can't use the utils in 15.0? I don't remember (same for other steps)

Reply:
Yes, but as far as I remember, I avoided them since the `utils.dragNDrop()` is not handling the case where a snippet can be dropped in the footer (as for `Form` blocks). So it always targets the footer to have the snippets added in the end of the page… I updated the code to use utils, with a small tweak to fix `dragNDrop()` :+1:

Reply:
**[EDIT]:** @qsm-odoo The answer here is not 100% correct. Actually, it seems that using `utils` in the `website_form_editor.js` file will trigger a `"missing dependencies: ['website.tour_utils']"` error (which explains the runbot failure)... After a discussion with LOCO (who had the [same issue previously](https://github.com/odoo/odoo/pull/140058#pullrequestreview-1755946880)), it seems to be caused by the "form submit" tours (`website_form_contactus_submit`, `website_form_editor_tour_submit`...) started without (or with "portal") login... `utils` should work if we split the tour here in a separate file, but I think it's better to restore the old version and use `utils` in FW (starting from `16.0`, the tours causing the problem were moved to a separate file `website_form_editor_frontend.js` and the code was adapted to use utils for the others :+1:)... What do you think?

Reply:
Alright :+1: